Sources/HipMobileUI/Views/BottomSheetView.cs

```diff
@@ -81,6 +90,25 @@ protected override void OnSizeAllocated(double width, double height)
             base.OnSizeAllocated(width, height);
         }
 
+        /// <summary>
+        /// Sets the position of the button afterwards for all android versions below Lollipop, where the calculated fabSize is not correct.
+        /// This repositioning cannot be done in the OnSizeAllocated method because the button have to be 
+        /// added first to get the right button width and height.
+        /// </summary>
+        private void SetButtonPosition(double width)
+        {
+            Button.SizeChanged += (sender, e) =>
+            {
+                Rectangle buttonRect = new Rectangle
+                {
+                    Top = Height - bottomSheetdSize - Button.Height / 2,
+                    Size = new Size(Button.Width, Button.Height),
+                    X = width * 0.9 - Button.Width
```

Comment:
Why do you multiply by `0.9`? Is this to introduce some padding on the right side of the button? If yes, why should the padding grow with the screen width? I'd suggest subtracting something between 4-8 dp, so e.g. something like `X = width - Button.Width - 4 * Resources.System.DisplayMetrics.Density`.

Reply:
I have placed the button in the same way it is placed on android versions below Lollipop ( see line 83 in BottomSheetView.cs). I think this was done before to set the button not to far to the right side. I can change this with a constant dp value, but the conversion from dp to pixels can lead to a strange button placement again.

Reply:
Can you try and position it as on >= Lollipop using a dp-constant padding? Otherwise on large screens, the button will have more space to its right side than on small screens. Actually, since `width` is a value in pixels, the screen doesn't even have to be large, it is sufficient to have a high-density display to move the button more to the left, right?

Reply:
I have added a constant dp value and it looks good on both android versions (< & >= Lollipop). It does not have exactly the same position on every device, but I think this shouldn't be a problem.